Frameworks/CoreGraphics/CGContext.mm

```diff
@@ -369,6 +377,30 @@ inline bool ShouldDraw() {
         return CurrentGState().ShouldDraw();
     }
 
+    inline void PushBeginDraw() {
+        if ((_beginEndDrawDepth)++ == 0) {
+            deviceContext->BeginDraw();
+        }
+    }
+
+    inline void PopEndDraw() {
+        if (--(_beginEndDrawDepth) == 0) {
+            THROW_IF_FAILED(deviceContext->EndDraw());
+        }
+    }
+
+    inline void EscapeBeginEndDrawStack() {
+        if ((_beginEndDrawDepth > 0) && ((_escapeBeginEndDrawDepth)++ == 0)) {
+            THROW_IF_FAILED(deviceContext->EndDraw());
```

Comment:
i actually already made the change anyway

---
In reply to: [97697849](https://github.com/Microsoft/WinObjC/pull/1750#discussion_r97697849) [](ancestors = 97697849,97696881,97696710,97695158,97468410)